src/ex2/ItemManager.js

```diff
@@ -0,0 +1,33 @@
+class ItemManager {
+    constructor() {
+        this.todoList = [];
+    }
+    
+    async addItem(todo){
+        if (!/^[0-9]+$/.test(todo))
+        {
+            return {id:this.todoList.length ,text:todo};
+        }
+        else{
+            const response = await (pokemon.getPokemonNameById(todo))
+            /*if(this.todoList.includes(response)) 
+            {
+                alert('The task in the listTodo') 
+                return;
+            }*/
+            return {id:this.todoList.length ,text:response};
+        }
+    }
+    async addArrayItem(todoArray)
+    {
+        todoArray = todoArray.filter((item) => item!=',');
```

Comment:
to split a string by commas you should use the .split() method. 
Please note 2 bugs that exist now:
<img width="489" alt="image" src="https://user-images.githubusercontent.com/36815567/170688515-736d2706-6bf9-42a1-9318-4b8000e27ffc.png">
todos with commas inside them are separated to different items

adding multiple numbers with spaces between them only adds the first pokemon
<img width="505" alt="image" src="https://user-images.githubusercontent.com/36815567/170688810-6a6c30ae-4adc-48f3-a684-a30515bac0fe.png">

use the trim() method to delete trailing spaces